### PR TITLE
Explain the limitations of the custom messages in UniqueEntity

### LINKED ISSUE #6929: UniqueEntity message can't include the value
Constraint messages can include `{{ value }}` in the `message` option to display the wrong value introduced by the user. Apparently, you can't do the same in the `UniqueEntity` constraint, so it should be documented here: http://symfony.com/doc/current/reference/constraints/UniqueEntity.html#message

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #6929. Thanks goes to @HeahDude, who made all the "detective work" to find this limitation.

### CODE REVIEW

File: reference/constraints/UniqueEntity.rst [hunk 1]
```diff
@@ -132,7 +132,11 @@ message
 
 **type**: ``string`` **default**: ``This value is already used.``
 
-The message that's displayed when this constraint fails.
+The message that's displayed when this constraint fails. When using multiple
+fields, the ``{{ vallue }}`` parameter only contains the value of the first
```
Comment (HeahDude) [inline reference/constraints/UniqueEntity.rst]: actually `{{ value }}` represents the entity, but only the first invalid field will trigger a violation to its path

**Comment by HeahDude** [comment]
Wonderful 👍 Thanks!

Status: reviewed

**Comment by OskarStark** [comment]
👍

**Comment by xabbuh** [comment]
This was first included in Symfony 3.1, wasn't it? I would also add a `versionadded` directive:

```rst
.. versionadded:: 3.1
    The ability to include the invalid value into the message was introduced
    in Symfony 3.1.
```

Status: Needs work

**Comment by javiereguiluz** [comment]
@xabbuh I've finally added the missing `versionadded` directive. Thanks!

**Comment by xabbuh** [comment]
👍 

Status: Reviewed

**Comment by wouterj** [comment]
Thanks for documenting this new feature Javier! I've merged this into 3.1, because this feature was introduced in 3.1 according to Christian.

**Pull request closed without merging**